samples/subsys/ipc/openamp/README.md

```diff
@@ -0,0 +1,72 @@
+OpenAMP Example Application
+===========================
```

Comment:
Our Zephyr doc standards use ### underlines for H1 headings, *** for H2, === for H3, and --- for H4.
So use ### underlines for this heading